constants/layouts.go

```diff
@@ -0,0 +1,101 @@
+package constants
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+type Layout struct {
```

Comment:
__[golint]__
* [golint] exported type Layout should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->